tests/Sniffs/PHP/data/shortListErrors.fixed.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+[$a] = foo();
+[$a, $b] = foo();
+[
+	[$a, $b],
+	$c,
+] = foo();
+[
+	$a,
+	[
+		$b,
+		[$c],
+		$d,
+	],
+] = foo();
+[
+	'a' => [
+		'b' => [$b],
+	],
+] = foo();
```

Comment:
PhpStorm is confused by the block above, but it is valid: https://3v4l.org/dAEfH.